ruby/engine/embedded_help.rb

```diff
@@ -36,7 +36,7 @@ def self.get_absolute_path(p)
       end
       absolute_path = ':' + absolute_path
     else
-      absolute_path = File.expand_path p2
+      absolute_path = File.expand_path p
```

Comment:
Weirdly enough, this has always been broken. p2 isn't defined here.

Reply:
correct. We don't use it directly basically, and checks were doing prior to calling it.